<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>COCOMO</string>
  </property>
  <widget class="QComboBox" name="comboBox_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RELY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>DATA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CPLX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TIME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>STOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VIRT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TURN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ACAP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>AEXP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PCAP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VEXP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LEXP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>MODP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TOOL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>39</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SCED</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_14">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_15">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>Очень низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Низкий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Номинальный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Высокий</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Очень высокий</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="sizeEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SIZE (KLOC)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_16">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Обычный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Промежуточный</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Встроенный</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Режим проекта</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
